add a non-LOTS subspace of R (#646)
pi-base/data: spaces/S000198/properties/P000017.md

---
space: S000198
property: P000017 #sigma-compact
value: true
---

> This space is a countable union of {P17} spaces

Reviewer: "This space is the union of ..." is simpler and good enough.

Author: ```suggestion
This space is the union of {P17} spaces
```
